autoload/neomake/signs.vim

```diff
@@ -235,8 +235,8 @@ function! neomake#signs#DefineHighlights() abort
 
     for [group, fgs] in items({
                 \ 'NeomakeErrorSign': [
-                \   neomake#utils#GetHighlight('Error', 'bg'),
-                \   neomake#utils#GetHighlight('Error', 'bg#')],
+                \   neomake#utils#GetHighlight('Error', 'fg'),
+                \   neomake#utils#GetHighlight('Error', 'fg#')],
```

Comment:
I see that `Error` is not even defined by default (using `nvim -u NONE` and then `:verb hi Error`), but `ErrorMsg` would be.
I based this on what the solarized theme gives me (where it is reversed):
> Error          xxx cterm=reverse ctermfg=1 guifg=White guibg=Red

So I think this PR should be changed to use `ErrorMsg` and `fg*`.